model/README.md

```diff
@@ -0,0 +1,90 @@
+# SyNAP Model Export
+This toolkit provides a convenient way of exporting models to the SyNAP format.
+The following models are currently supported:
+
+| Model | Export Script |
+|-------|---------------|
+| [YOLOv8](https://docs.ultralytics.com/models/yolov8/) | [export/yolo.py](/model/export/yolo.py) |
```

Comment:
What are the limitations here to generalize this tool for any version of YOLO model?

Reply:
The current limitation is that the model export has to happen through the ultralytics package, specifically through `Model.export()`. So we can't do, for example, YOLOv5 as that's it's own repository and not through the ultralytics package.
That being said, the model exporter is modular. So someone could just subclass the base `ModelExporter` and provide their own `export_model()`  for any model. All they need to ensure is that `export_model()` should return the path to the exported tflite/onnx/torch etc. model.